packages/v4/src/content/design-guidelines/components/drag-and-drop/drag.md

```diff
@@ -0,0 +1,72 @@
+---
+id: Drag and drop
+section: components
+---
+The **drag and drop** interaction can be used to reposition elements on screen into a layout that benefits the user. This gives the user more flexibility to arrange and/or group items without having to make code changes.
+
+## Usage
+### Simple list
+1. **Draggable area:** The drag and drop interaction can be triggered with click and hold over any part of the item. The `fa-grip` icon is used to show that the drag and drop interaction is available.
+
+<img src="./img/Simple-list-drag-1.png" width="588" alt="Showing the area in which an item can be dragged."/>
+
+2. **Bounding box:** Upon click & hold a `pf-color-blue-400` border will show the draggable area that is available. 
+
+3. **onDrag event:** The list item being dragged will also use a `pf-color-blue-400` border to highlight it as the item being dragged and all other list items will switch to a disabled state. The space where the item is being dragged from will remain empty to indicate its original position in the list.
+
+<img src="./img/Simple-list-drag-2-3.png" width="683" alt="Borders appear around the item and bounding box shows the area in which an item can be dragged."/>
+
+4. **postDrag event:** Once dropped, the items will be reordered based on the user’s action. The space left empty is then filled by the next item in the list.
+
+<img src="./img/Simple-list-drag-4.png" width="588" alt="postDrag drops item into new position and all border highlights are disabled."/>
+
+5. **Error state:** If the list item is dragged outside the bounding box the borders on the dragged item will switch to `pf-color-red-100` and the cursor will change to an error state to indicate an invalid placement. If the user releases the cursor outside the bounding area the dragged item will return to its default position.
```

Comment:
```suggestion
5. **Error state:** If the list item is dragged outside the bounding box the borders on the dragged item will switch to `--pf-global--danger-color--100` and the cursor will change to an error state to indicate an invalid placement. If the user releases the cursor outside the bounding area the dragged item will return to its default position.
```

Reply:
"Error" state is kind of generic. It works, but FWIW we use a modifier of `.pf-m-drag-outside` to apply the styling for when you're dragging over an area that isn't droppable.